build/Tests/Benchmark/Framework.Benchmark.vcxproj

```diff
@@ -221,7 +221,9 @@
   </ItemGroup>
   <ItemGroup>
     <ClangCompile Include="$(StarboardBasePath)\tests\Benchmark\BenchmarkSampleTests.mm" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\Benchmark\KVOBenchmarkTests.mm" />
```

Comment:
That app is not used at all (it is a manual tests).  When it was written, we didn't have the benchmark tests.  It should be safe to remove *once* all the pieces of it are salvaged.  

---
In reply to: [108012483](https://github.com/Microsoft/WinObjC/pull/2323#discussion_r108012483) [](ancestors = 108012483)